system/lib/gl/libprocaddr.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+// GL proc address library integration
+
+extern void* emscripten_GetProcAddress(const char *name);
+
+void* SDL_GL_GetProcAddress(const char* name) {
+  return emscripten_GetProcAddress(name);
+}
+
+void* eglGetProcAddress(const char* name) {
+  return emscripten_GetProcAddress(name);
+}
+
+void* glfwGetProcAddress(const char* name) {
+  return emscripten_GetProcAddress(name);
+}
```

Comment:
Would it not make sense to these to live in their various different libraries?  

Is `emscripten_GetProcAddress` just for GL symbols?   I wonder if it should be renamed to reflect that?

Reply:
We don't have compiled libraries for them, and adding new ones just for what are basically aliases seemed like overkill Perhaps we can even just make them aliases in the headers, if you prefer that?

Yes, that function is just for GL symbols. I'm not opposed to renaming it (separate from this PR I assume?)

Reply:
I'm find with this now if tests pass.